src/App/Pages/Accounts/TwoFactorPage.xaml

```diff
@@ -16,14 +16,18 @@
     <ContentPage.ToolbarItems>
         <ToolbarItem Text="{u:I18n Cancel}" Clicked="Close_Clicked" Order="Primary" Priority="-1"
                      x:Name="_cancelItem" />
-        <ToolbarItem Text="{u:I18n Continue}" Clicked="Continue_Clicked" Order="Primary"
-                     x:Name="_continueItem" />
     </ContentPage.ToolbarItems>
 
     <ContentPage.Resources>
         <ResourceDictionary>
             <u:InverseBoolConverter x:Key="inverseBool" />
             <u:IsNullConverter x:Key="isNull" />
+            <ToolbarItem Icon="more_vert.png" Clicked="More_Clicked" Order="Primary"
+                x:Name="_moreItem" x:Key="moreItem"
+                 AutomationProperties.IsInAccessibleTree="True"
+                 AutomationProperties.Name="{u:I18n Options}" />
+            <ToolbarItem Text="{u:I18n UseAnotherTwoStepMethod}" Clicked="Methods_Clicked" Order="Secondary"
+                x:Name="_useAnotherTwoStepMethod" x:Key="useAnotherTwoStepMethod" />
```

Comment:
Formatting nit: please make this consistent with how other elements are layed out, with each attribute on a new line:
```suggestion
            <ToolbarItem Text="{u:I18n UseAnotherTwoStepMethod}" 
                Clicked="Methods_Clicked" 
                Order="Secondary"
                x:Name="_useAnotherTwoStepMethod" 
                x:Key="useAnotherTwoStepMethod" />
```